src/graphql/mutations.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  GraphQLID,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  Source
+} from "graphql";
+import {GraphqlObjectTypeFields} from "./utils";
+
+const createTypeMutation = (
+  type: GraphQLObjectType,
+  typeName: string,
+  fields: GraphqlObjectTypeFields,
+  createCallback: (args: any) => Promise<any>
+) => {
+  return {
+    type: type,
+    description: 'Creates a new ' + typeName,
+    args: fields,
+    resolve: async (_: Source, ...rest: any) => {
+      console.log("Executing mutation create from " + typeName);
+
+      return createCallback(rest)
+    }
+  }
+}
+
+const updateTypeMutation = (
+  type: GraphQLObjectType,
+  typeName: string,
+  fields: GraphqlObjectTypeFields,
+  updateCallback: (id: string, args: any) => Promise<any>
+) => {
+  return {
+    type: type,
+    description: 'Updates a ' + typeName,
+    args: fields,
+    resolve: async (_: Source, { id, ...rest }: any) => {
+      console.log("Executing mutation update from " + typeName);
+
+      return updateCallback(id, rest)
+    }
+  }
+}
+
+const deleteTypeMutation = (
```

Comment:
Podemos tener como convenció no nombrar funciones con sustantivos (`buildDeleteTypeMutation`) ?

Reply:
https://hexlet.io/blog/posts/perfect-code-naming-mistakes-to-avoid-in-programming-i#function-as-a-noun